app/models/form/sales/pages/about_deposit_without_discount.rb

```diff
@@ -0,0 +1,31 @@
+class Form::Sales::Pages::AboutDepositWithoutDiscount < ::Form::Page
+  def initialize(id, hsh, subsection)
+    super
+    @header = "About the deposit"
+    @description = ""
+    @subsection = subsection
+    @depends_on = [
+      { "type" => 2 },
+      { "type" => 24 },
+      { "type" => 16 },
+      { "type" => 28 },
+      { "type" => 31 },
+      { "type" => 30 },
+      { "type" => 8 },
+      { "type" => 14 },
+      { "type" => 27 },
+      { "type" => 9 },
+      { "type" => 29 },
+      { "type" => 21 },
+      { "type" => 22 },
+      { "type" => 10 },
+      { "type" => 12 },
```

Comment:
Is it possible for this to be a method on the sales_log model?
```
@depends_on = [{is_type_discount? => false}]
```

and in the model 
```
def is_type_discount?
    type == 18
end
```
And could use it in the other question as ` [{is_type_discount? => true}]`

Reply:
Idk if there's a better name for this

Reply:
I don't think that's possible - I think depends_on has to be an array with a db field and value otherwise it won't get parsed correctly

Reply:
Hmm, I thought we were doing methods on interruption screens, for example:
```
"depends_on": [
                {
                  "female_in_pregnant_household_in_soft_validation_range?": true
                }
              ],
```

Reply:
Happy for you to merge as is tho

Reply:
hm ok I'll have another go, thanks! Will merge either way

Reply:
Got it, cheers!